services/api/schemas/events.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+from typing import Literal, Tuple
+from pydantic import BaseModel
+
+class Event(BaseModel):
+    sample_id: int
+    created: str = datetime.now().isoformat()
+    validated: bool = False
+    
+class Point(Event):
+    time: int
+
+class Region(Event):
+    time_min: float
+    time_max: float
+    
+class UFO(Point):
```

Comment:
I think `UFO` `ELMRegion` and `ELM` are too specific. I think we need to stick to generic types, more like:

`TimeSeriesRegion`
`TimeSeriesPoint`
`BoundingBox`

Each may have a `label` associated with it, which could be flexible.